### ISSUE TITLE
Do more augmentations

### ISSUE THREAD

**Issue by gsganden** [UptakeOpenSource/autofocus#58]
Given that the animals often occupy only a small part of the image, we may get better performance with a lot of augmentations involving high zoom. This kind of approach is often used with medical images, e.g. where a tumor might occupy a small part of the image.

**Comment by aminzarshenas** [comment]
Hey @gsganden. Is this still a feature you're looking for? I would like to help with that.

**Comment by gsganden** [comment]
Yes, that would be great. See https://github.com/uptake/autofocus/blob/master/autofocus/train_model/train_multilabel_model.ipynb for a record of the most recent training run.
